client/components/CodeMirror.js

```diff
@@ -28,6 +31,7 @@ var CodeMirror = React.createClass({
     return (
```

Comment:
insert a blank line right about the return to separate the variable declaration from it (between lines 30 and 31)